player/src/main/java/net/pubnative/player/VASTPlayer.java

```diff
@@ -324,7 +325,11 @@ private void setEmptyState() {
         mQuartile = 0;
         mTrackingEventMap = null;
         mProgressTracker = null;
-        mBannerBitmap = null;
+        if (mIsBannerSet) {
```

Comment:
but what if developer doesn't set banner for 2nd ad (I mean set to null)? for our case yes we can do this from FeedVideo or Video.